phases/register.go

```diff
@@ -1,9 +1,119 @@
 package phases
 
-import "fmt"
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
 
-// Register ...
-func Register(progress Progress) error {
-	fmt.Println("Register")
+	"github.com/bitrise-io/go-utils/log"
+)
+
+var baseURL = "https://app.bitrise.io"
+
+var (
+	webhookAttemptCount int
```

Comment:
This should be in the scope of the place where it is used, also that would mean you no need to reset it in line 30